.circleci/config.yml

```diff
@@ -39,7 +39,12 @@ jobs:
       - run:
           name: Build docker image
           command: |
-            docker build --cache-from=${IMAGE_NAME} -t ${IMAGE_NAME} -t ${IMAGE_NAME}:$(cat VERSION) .
+            docker build \
+              --cache-from=${IMAGE_NAME} \
+              --build-arg BUILD_DATE=`date -u +"%Y-%m-%dT%H:%M:%SZ"` \
```

Comment:
Please don't use backticks for creating subshells use 
```
--build-arg BUILD_DATE=$(date -u +"%Y-%m-%dT%H:%M:%SZ)
```

For date I think it would be more readable to specify the format so use `date --rfc-3339=s` instead of your date command.

Reply:
DONE